Resources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>NSPrivacyCollectedDataTypes</key>
+        <array>
+            <dict>
+                <key>NSPrivacyCollectedDataType</key>
+                <string></string>
+                <key>NSPrivacyCollectedDataTypeLinked</key>
+                <false/>
+                <key>NSPrivacyCollectedDataTypeTracking</key>
+                <false/>
+                <key>NSPrivacyCollectedDataTypePurposes</key>
+                <array>
+                    <string></string>
+                </array>
+            </dict>
+        </array>
+        <key>NSPrivacyAccessedAPITypes</key>
+        <array>
+            <dict>
+                <key>NSPrivacyAccessedAPIType</key>
+                <string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
+                <key>NSPrivacyAccessedAPITypeReasons</key>
+                <array>
+                    <string>3B52.1</string>
```

Comment:
@weichsel From a code search, ZIPFoundation calls stat() (a required reason API) in FileManager+ZIP.swift. However, it does not call UIDocumentPickerViewController or UIDocumentBrowserViewController, and it does not reference UIKit anywhere, so it looks like ZIPFoundation does not specifically request users to grant access to timestamps of files or directories.

Users may grant access to access timestamps of files or directories in an app that integrates with ZIPFoundation, but that would be the app's responsibility to declare 3B52.1, instead of ZIPFoundation.

Could you elaborate why do you declare 3B52.1 for ZIPFoundation? Or would you consider removing it?